src/main/java/nextstep/controller/ReservationController.java

```diff
@@ -13,33 +15,33 @@
 @RestController
 @RequestMapping("/reservations")
 public class ReservationController {
+    private final ReservationService reservationService;
+
     @Autowired
-    private WebAppReservationService webAppReservationService;
+    public ReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
 
     @PostMapping()
-    public ResponseEntity createReservation(@RequestBody CreateReservationDTO reservationDto) {
-        long id = webAppReservationService.addReservation(reservationDto);
-        if (id == -1) {
+    public ResponseEntity createReservation(@RequestBody CreateReservationDto reservationDto) {
+        try {
+            long id = reservationService.addReservation(reservationDto);
+            return ResponseEntity.created(URI.create("/reservations/" + id)).build();
+        } catch (DuplicateTimeReservationException ex) {
```

Comment:
`DuplicateTimeReservationException`, `IllegalReservationTimeException` 두 개의 `exception` 모두 response 가 동일한데 한 곳에 작성해주는 것은 어떨까요?
```suggestion
        } catch (DuplicateTimeReservationException | IllegalReservationTimeException ex) {
```

Reply:
한 곳에 작성하려고 시도했었는데 형식을 착각했었네요...
DuplicateTimeReservationException ex | IllegalReservationTimeException ex2
이런 식으로 잘못 작성해서 빨간 줄 뜨길래 자바 문법이 아니었나 착각했네요 ㅠ

작성해주신 방법대로 수정해보겠습니다!